prime-dotnet-webapi/Services/CommunitySiteService.cs

```diff
@@ -108,6 +116,13 @@ private static void GroupSitesToOrgVisually(PaginatedList<CommunitySiteAdminList
             }
         }
 
+        private async Task<int> GetDuplicatePecCount(int? careSettingCode, string pec, int orgSiteId)
```

Comment:
Why is this named `orgSiteId`?

Reply:
oh, meant to be original site id (not PEC).

Reply:
Updated the name to be clear